recipes/caffe/meta.yaml

```diff
@@ -0,0 +1,117 @@
+{% set base_version = "1.0.0" %}
+{% set version_postfix = "rc3" %}
+{% set version = base_version + version_postfix %}
+
+package:
+  name: caffe
+  version: {{ version }}
+
+source:
+  fn: caffe_{{ version }}.tar.gz
+  url: https://github.com/BVLC/caffe/archive/{{ version_postfix }}.tar.gz
+  sha1: 79e075d06bea856164c926b92a40242b0369965b
+
+build:
+  number: 0
+  skip: true  # [not (linux and py27)]
+
+requirements:
+  build:
+    - cmake
+    - cython
+    - nomkl
+    - openblas
+    - boost
+    - hdf5
+    - gflags
+    - glog
+    - h5py
+    - hdf5
+    - ipython
+    - leveldb
+    - lmdb
+    - matplotlib
+    - networkx
+    - nose
+    - numpy
+    - opencv
+    - pandas
+    - pillow
+    - protobuf
+    - python
+    - python-dateutil
+    - python-gflags
+    - python-leveldb
+    - pyyaml
+    - scikit-image
+    - scipy
+    - six
+    - snappy
+
+  run:
+    - nomkl
+    - openblas
+    - boost
+    - hdf5
+    - gflags
+    - glog
+    - h5py
+    - hdf5
+    - ipython
+    - leveldb
+    - lmdb
+    - matplotlib
+    - networkx
+    - nose
```

Comment:
Though we are testing during the build so I think this is still needed here. However, I removed it from the run requirements where it was before.